smf/sled-agent/gimlet-standalone/config-rss.toml

```diff
@@ -4,8 +4,9 @@
 # Agent API.  See the `RackInitializeRequest` type in bootstrap-agent or its
 # OpenAPI spec (in openapi/bootstrap-agent.json in the root of this workspace).
 
-# The /56 subnet for the rack.
-# Also implies the /48 AZ subnet.
+# The /56 subnet for this rack.  This subnet is internal to the rack and fully
+# managed by Omicron, so you can pick anything you want.  The rack-specific /56
```

Comment:
nit: pick anything you want -> pick anything you want within the IPv6 ULA range. As a heads up, this configuration option will be going away entirely soon, as the control plane will decide this for itself without user input.

Reply:
Done in 2fe16aa9848d7bed601255980936a17bd4e911c3.